### PR TITLE
#5664 follow up visits

### PR DESCRIPTION
**Pull request by horatiu-cosma**
Fixes #

### CODE REVIEW

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/ContactDirectoryPage.java [hunk 1]
```diff
@@ -42,4 +42,10 @@
   public static final By LAST_NAME_COLUMNS = By.cssSelector("[role=rowgroup] tr>td:nth-child(9)");
   public static final By TYPE_OF_CONTACT_COLUMNS =
       By.cssSelector("[role=rowgroup] tr>td:nth-child(11)");
+  public static final By FOLLOW_UP_VISITS_SPAN =
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/ContactDirectoryPage.java:45]: please rename SPAN and change with what type of element is in FE

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+public class CreateNewVisitPage {
+  public static final By PERSON_AVAILABLE_AND_COOPERATIVE =
+      By.cssSelector("#visitStatus > span > label");
+  public static final By DATE_AND_TIME_OF_VISIT_INPUT = By.cssSelector("#visitDateTime_date input");
+  public static final By TIME_OF_VISIT_INPUT = By.cssSelector("#visitDateTime_time div");
+  public static final By VISIT_REMARKS_INPUT = By.id("visitRemarks");
+  public static final By CURRENT_BODY_TEMPERATURE_COMBOBOX = By.cssSelector("#temperature div");
+  public static final By SOURCE_BODY_TEMPERATURE_COMBOBOX =
+      By.cssSelector("#temperatureSource div");
+  public static final By CHILLS_OR_SWATS_LABEL = By.cssSelector("#chillsSweats > span label");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java:32]: same with LABEL

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      By.cssSelector("#acuteRespiratoryDistressSyndrome > span label");
+  public static final By COUGH_LABEL = By.cssSelector("#cough > span label");
+  public static final By DIFFICULTY_BREATHING_LABEL =
+      By.cssSelector("#difficultyBreathing > span label");
+  public static final By SORE_THROAT_PHARYNGITIS_LABEL = By.cssSelector("#soreThroat > span label");
+  public static final By FAST_HEART_RATE_LABEL = By.cssSelector("#fastHeartRate > span label");
+  public static final By DIARRHEA_LABEL = By.cssSelector("#diarrhea > span label");
+  public static final By NAUSEA_LABEL = By.cssSelector("#nausea > span label");
+  public static final By LOSS_OF_SMELL_LABEL = By.cssSelector("#lossOfSmell > span label");
+  public static final By LOSS_OF_TASTE_LABEL = By.cssSelector("#lossOfTaste > span label");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java:56]: same with all labels above

**Review by Mihai-Bota — APPROVED** [review]

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.pages.application.contacts;
+
+import org.openqa.selenium.By;
+
```
Comment (alinapopaiustina) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java:22]: Missing the type for more identifiers in the page

**Review by alinapopaiustina — APPROVED** [review]

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java (hunk 4 above)
Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/CreateNewVisitPage.java:22]: what do you mean? All locators from all POs are By locators, even if we use css, xpath etc, the import is the same

**Pull request merged**